### DISCUSSION TITLE
wiki/pyqgis-sd/06_package/pyinstaller

### DISCUSSION THREAD

**Discussion by giscus** [SX-Code/blog-comments#19] [Announcements]
# wiki/pyqgis-sd/06_package/pyinstaller

敬请期待

https://sx-code.github.io/wiki/pyqgis-sd/06_package/pyinstaller.html

**Comment by wiaini** [comment]
大佬赶紧更新啊

**Reply by SX-Code** [reply]
已更新，注意查收😊

**Reply by wiaini** [reply]
太棒了！！

**Comment by whh-meteor** [comment]
大佬想请教一下为什么，打包后的exe在本机可以执行，在其他机器上就报错： Traceback (most recent call last):
  File "F:\Desktop\QGIS_Tools\main.py", line 48, in <module>
    from qgis._core import QgsApplication
ImportError: DLL load failed while importing _core: 找不到指定的模块。  应该怎么处理这个问题 ？

**Reply by SX-Code** [reply]
那肯定是环境问题，你看一下打包时是不是有某个模块没放进去

**Reply by whh-meteor** [reply]
感谢回复。解决方法参考： QGIS 3.40，Python312 ， Pyinstaller 6.12 无论怎么打包都找不到，甚至手动拷贝了相关的依赖，也不行 。 最后的解决办法是降级PyInstaller的版本到5.X 。 一次打包成功并可以在其他机器上运行。

**Comment by whh-meteor** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
